app/assets/stylesheets/homepage.css.scss

```diff
@@ -6,7 +6,11 @@
 #nav-bar {
     position:absolute;
     background-color: rgba(0,0,0,.4);
-    padding: 10px;
+    padding-top: 10px;
+}
+
+#nav-bar .mdl-layout__header-row {
```

Comment:
Avoid using id selectors<br>Merge rule `#nav-bar .mdl-layout__header-row` with rule on line 6<br>Selector `mdl-layout__header-row` should be written in lowercase with hyphens